sample/SampleServer/Program.cs

```diff
@@ -35,8 +36,10 @@ static async Task MainAsync(string[] args)
                 options
                     .WithInput(Console.OpenStandardInput())
                     .WithOutput(Console.OpenStandardOutput())
-                    .ConfigureLogging(x => x.AddSerilog())
-                    .AddDefaultLoggingProvider()
+                    .ConfigureLogging(x => x
+                        .AddSerilog()
+                        .AddLanguageServer(LogLevel.Error)
+                        .SetMinimumLevel(LogLevel.Error))
```

Comment:
why do we need to pass in the `LogLevel` for both the `AddLanguageServer` and `SetMinimumLevel`?

Reply:
You don't need to. It's an option. One is global, the other one is just for LSP logging.

Reply:
@mholo65 I gave this a go:

```
.AddSerilog()
.AddLanguageServer()
.SetMinimumLevel(LogLevel.Error)
```

and the LSP side was only giving Information and Error.